.github/workflows/ci.yml

```
@@ -86,6 +86,12 @@ jobs:
     name: Test (${{ matrix.os }}, ${{ matrix.node }})
     runs-on: ${{ matrix.os }}
 
+    env:
+      # Persisted V8 compile cache (restored/saved by the "Cache Node compile
+      # cache" step). Set at job level so vitest workers and any forked child
+      # processes inherit NODE_COMPILE_CACHE.
+      NODE_COMPILE_CACHE: ${{ runner.temp }}/node-compile-cache
```

actionlint (1.7.12)

[error] context "runner" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)